### DISCUSSION TITLE
Privacy options for Profile

### DISCUSSION THREAD

**Discussion by deadpahn** [RetroAchievements/RAWeb#971] [Ideas]
Hello,

It would be great to have better privacy options on the profile. For my own record I think it's cool to have a time stamp but I don't want "Played Mario Paint at 3:45am on Friday Night" to be seen by the public lol. I realize this might mess with the global boards but I'm not trying to be the next billy mitchel. I wouldn't mind opting out of leaderboards if it meant I could control privacy settings. Some things I don't mind sharing with friends (basic trophy data) but I dunno, the time stamp thing is weirding me out a bit.

**Comment by StingX2** [comment]
It'd require an opt out for good part of the leaderboards aka softcore mode. Timestamps are needed to figure out cheaters.

**Reply by deadpahn** [reply]
Sure, I don't mind if it's logged for moderators, but I don't want my girlfriend seeing I decided to play Mario RPG instead of going to dinner with her parents, know what I mean?

**Reply by meleu** [reply]
Also I don't want my employer to know I was playing [Zooming Secretary](https://retroachievements.org/game/4650) while in the night shift...

![homer-at-work](https://user-images.githubusercontent.com/8508804/166812509-1a996cd9-2b61-43d6-a08b-5a8b7f69457f.png)

The current solution we have for this is to ask your account to be Untracked: https://docs.retroachievements.org/Global-Leaderboard-and-Achievement-Hunting-Rules/#untracked

**Reply by deadpahn** [reply]
Thanks!

I'll have a look for that.
